config/prow/cluster/cherrypick_deployment.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cherrypick
+  namespace: prow
+spec:
+  selector:
+    app: prow
+    component: cherrypick
+  ports:
+  - port: 80
+    targetPort: 8888
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cherrypick
+  labels:
+    app: prow
+    component: cherrypick
+  namespace: prow
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: prow
+      component: cherrypick
+  template:
+    metadata:
+      labels:
+        app: prow
+        component: cherrypick
+    spec:
+      containers:
+      - name: cherrypick
+        image: gcr.io/k8s-prow/cherrypicker:v20210305-64a6d4d83a
+        args:
+        - --github-token-path=/etc/github/token
+        - --github-endpoint=http://ghproxy
+        - --github-endpoint=https://api.github.com
+        - --dry-run=false
+        ports:
+          - name: http
+            containerPort: 8888
+        volumeMounts:
+        - name: hmac
+          mountPath: /etc/webhook
+          readOnly: true
```

Comment:
Where is this used?

Reply:
I explicitly see this file not being used by the container.
I was thinking it might be used by the deployment at some stage as I do not know the internals of  `gcr.io/k8s-prow/cherrypicker `image.

Reply:
lets remove and see how it goes,

Reply:
@mkumatag I get the below error when I remove `hmac` volume and corresponding mount:
```
{"error":"error reading /etc/webhook/hmac: open /etc/webhook/hmac: no such file or directory","file":"prow/external-plugins/cherrypicker/main.go:96","func":"main.main","level":"fatal","msg":"Error starting secrets agent.","time":"2021-03-21T12:08:42Z"}
```
Seems the code from below file needs the webhook secret details:
`https://github.com/kubernetes/test-infra/blob/master/prow/external-plugins/cherrypicker/main.go#L96`

Reply:
Okay..